.gitlab-ci.yml

```diff
@@ -190,7 +190,7 @@ android-armv7:
     - triggers
   script:
     - cargo build --target=armv7-linux-androideabi
-    # TODO: check that `arm-linux-androideabi-objdump -x ./target/armv7-linux-androideabi/release/parity | grep c++_shared` is empty
+    - $(arm-linux-androideabi-objdump -x ./target/armv7-linux-androideabi/debug/parity | grep c++_shared | wc -l) -eq 0
```

Comment:
I think this should work:

```bash
if [ $(arm-linux-androideabi-objdump -x ./target/armv7-linux-androideabi/debug/parity | grep c++_shared | wc -l) -ne 0]; then
   echo "FAIL!!"
fi
```
But I'm not really sure how this will be executed in docker?

Reply:
this option is better